File: VonexDealer.API/Models/Order/Internet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace VonexDealer.API.Models.Order
{
    public class Internet
    {
        public Int64 InternetID { get; set; }
        public Int64 OrderID { get; set; }
        public string InternetType { get; set; }
        public Int64 RatePlanID { get; set; }
        public string UsageAllowance { get; set; }
        public string SpeedRequested { get; set; }
        public decimal MonthlyPrice { get; set; }
        public decimal ConnectionFee { get; set; }
        public string ContractTerm { get; set; }
        public Int64 AddressID { get; set; }
        public string ServiceNumber { get; set; }
        public Int64 ContactID { get; set; }

    }
}

File: VonexDealer.Vue/ClientApp/models/Internet.js

export class Internet {
    constructor({ internetID = 0,
        orderID = 0,
        internetType = null,
        ratePlanID = 0,
        usageAllowance = 0,
        speedRequested = null,
        monthlyPrice = 0.00,
        connectionFee = 0.00,
        contractTerm = 0,
        addressID = null,
        serviceNumber = null,
        contactID = null
    } = {}) {
        this.internetID = internetID;
        this.orderID = orderID;
        this.internetType = internetType;
        this.ratePlanID = ratePlanID;
        this.usageAllowance = usageAllowance;
        this.speedRequested = speedRequested;
        this.monthlyPrice = monthlyPrice;
        this.connectionFee = connectionFee;
        this.contractTerm = contractTerm;
        this.addressID = addressID;
        this.serviceNumber = serviceNumber;
        this.contactID = contactID;
    }
}

export function createInternet(data) {
    return Object.freeze(new Internet(data));
}
